### PR TITLE
fix user location

### PR DESCRIPTION
**Pull request by Aliaksandrzaz**

### CODE REVIEW

File: js/src/features/human-resouce/components/EmployeeLocationEditForm.tsx [hunk 1]
```diff
@@ -23,6 +23,8 @@ interface Props {
   edit: (data: EditEmployeeLocationData) => void;
 }
 
+const DATE_FORMAT = "yyyy-MM-dd";
```
Comment (kwoon) [inline js/src/features/human-resouce/components/EmployeeLocationEditForm.tsx:26]: Try to use formatISO function. And i suggest you to use [date-fns fp branch](https://github.com/date-fns/date-fns/blob/master/docs/fp.md).

File: js/src/features/human-resouce/components/EmployeeLocationList.tsx [hunk 2]
```diff
@@ -20,15 +18,8 @@ export const EmployeeLocationList: React.FC<Props> = ({
...
-        label={`${item.startedAt} - ${item.endedAt}`}
-        color={
-          isWithinInterval(
-            { start: parseISO(item.startedAt), end: parseISO(item.endedAt) },
-            new Date(),
-          )
-            ? "green"
-            : "blue"
-        }
+        label={`${item.startedAt} - ${!!item.endedAt ? item?.endedAt : 'till now'}`}
```
Comment (kwoon) [inline js/src/features/human-resouce/components/EmployeeLocationList.tsx:21]: Maybe better to use `??` or `||` operator?

File: js/src/features/human-resouce/components/EmployeeLocationList.tsx [hunk 3]
```diff
@@ -20,15 +18,8 @@ export const EmployeeLocationList: React.FC<Props> = ({
...
+        color={!item.endedAt ? "green" : "blue"}
```
Comment (kwoon) [inline js/src/features/human-resouce/components/EmployeeLocationList.tsx:22]: In ternary or full for of if operator in condition always use positive value.
```js
// BAD
!item.endedAt ? "green" : "blue"

// GOOD
item.endedAt ? "blue" : "green"
```

File: js/src/features/human-resouce/components/LocationAssignForm.tsx [hunk 4]
```diff
@@ -16,7 +17,7 @@ interface Props {
   assign: (form: AssignLocationForm) => void;
 }
 
-const DATE_FORMAT = "YYYY-MM-DD";
+const DATE_FORMAT = "yyyy-MM-dd";
```
Comment (kwoon) [inline js/src/features/human-resouce/components/LocationAssignForm.tsx:20]: See above.

File: js/src/features/human-resouce/components/EmployeeLocationList.tsx (hunk 3 above)
Comment (Aliaksandrzaz) [reply js/src/features/human-resouce/components/EmployeeLocationList.tsx:22]: fix

File: js/src/features/human-resouce/components/EmployeeLocationList.tsx (hunk 2 above)
Comment (Aliaksandrzaz) [reply js/src/features/human-resouce/components/EmployeeLocationList.tsx:21]: fix

File: js/src/features/human-resouce/components/EmployeeLocationEditForm.tsx (hunk 1 above)
Comment (Aliaksandrzaz) [reply js/src/features/human-resouce/components/EmployeeLocationEditForm.tsx]: fix

File: js/src/features/human-resouce/components/LocationAssignForm.tsx (hunk 4 above)
Comment (Aliaksandrzaz) [reply js/src/features/human-resouce/components/LocationAssignForm.tsx]: fix

**Pull request merged**